src/test_helpers/gold_stream_evaluator.h

```diff
@@ -0,0 +1,49 @@
+#ifndef GOLD_STREAM_EVALUATOR_H_
```

Comment:
Please change `#define` guards to `<PROJECT>_<PATH>_<FILE>_H_` fashion.

Reply:
I am actually thinking about this. Assuming we have all the things in subdirectories of src, I guess with <PATH> as of current subdirectory name should be good enough. Previous files has sth like: `BART_AQDATA_AQ_BASE_H_`

Reply:
@weixiong-zheng-berkeley On it! In the guide they say
> For example, the file foo/src/bar/baz in project foo should have the following guard: `#ifndef FOO_BAR_BAZ_H_`

Should we leave out the `SRC` tag because most of ours will have it? So this one would be `BART_TEST_HELPERS_GOLD_STREAM_EVALUATOR_H_`?

Reply:
Keep SRC. As long as we are keeping tests folder for certain ctest cases, SRC is not the only place we are having source codes.